bookdown/07-special-spatiotemp.Rmd

```diff
@@ -84,6 +86,8 @@ rr_sp = resample(
 rr_sp$aggregate(measures = msr("classif.ce"))
 ```
 
+<!--JM: shouldn't we use `benchmark_grid()` to ensure that the same partitions are used for both resamplings?-->
```

Comment:
Shouldn't we use `benchmark_grid()` to ensure that the same partitions are used for both resamplings?

Reply:
`benchmark_grid()` is just a wrapper around `resample()`. I think calling the same function two times with just a different method is more clear for demonstration purposes. 

Why do you expect the partitions to be equal here? We are using different resampling methods so the partitions will be different.

Reply:
You are absolutely right, my bad!